### PR TITLE
Update entrypoint.sh add support for multi IP tunnel

### PR DESCRIPTION
**Pull request by Lukas0025**
This pull request adding support for tunneling over server what have more that one IP address. For example if server have 3 IP on 3 networks

```
net: 127.0.0.0/8                          ip: 127.0.0.1    (localhost)
net: 192.168.1.0/24                       ip: 192.168.1.2
net: 10.0.0.0/24                          ip: 10.0.0.2 
```
ssh syntax for -R

```
-R [bind_address:]port:host:hostport
```
When bind_address is omitted, the port is bound on the loopback interface (in my case). But i want forward to 192.168.1.2 => To fix it use bind address.
[from: [https://superuser.com/questions/588591/how-to-make-ssh-tunnel-open-to-public](https://superuser.com/questions/588591/how-to-make-ssh-tunnel-open-to-public)]

In new version use environment `SSH_BIND_IP`
```
SSH_BIND_IP=192.168.1.2
```

_in default is SSH_BIND_IP set to 0.0.0.0 it means forward to all IPs_

### CODE REVIEW

**Comment by jnovack** [comment]
I'm failing to see the point (my fault, not yours).    In the documentation, there are 4 terms defined, local, target, remote and source; can you please draw me a diagram (I like pretty pictures) of which endpoint has multiple IP addresses and the issue you are running into?

**Comment by Lukas0025** [comment]
ok. Here is simple sample.

ssh tunnel

I want make public my service on 192.0.0.1:80 with my server with IP 10.0.0.1 (this ip is not public) on port 8080. To create this i use this command `ssh -R 10.0.0.1:8080:192.0.0.1:80 root@203.0.113.10`

**More info:**
if you use `0.0.0.0 ` as bind address it forwards on all IP on remote server (it forwards from 10.0.0.1:8080 and 203.0.113.10:8080 too)
If you not set bind address it use always first interface (loopback can by first interface too)

**Comment by Lukas0025** [comment]
something new here?

**Comment by jnovack** [comment]
> something new here?

Yes, my personal life has a lot of new things going on. Am I allowed to sort them out before I deal with patches on Github?

**Comment by Lukas0025** [comment]
> Yes, my personal life has a lot of new things going on. Am I allowed to sort them out before I deal with patches on Github?

Yes of course. I was afraid that you did not notice the last comment due to the long inactivity for which I have not been accustomed to you.

File: entrypoint.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ fi
 # Pick a random port above 32768
 DEFAULT_PORT=$RANDOM
 let "DEFAULT_PORT += 32768"
-echo [INFO] Tunneling ${SSH_HOSTUSER:=root}@${SSH_HOSTNAME:=localhost}:${SSH_TUNNEL_REMOTE:=${DEFAULT_PORT}} to ${SSH_TUNNEL_HOST=localhost}:${SSH_TUNNEL_LOCAL:=22}
+echo [INFO] Tunneling over ${SSH_HOSTUSER:=root}@${SSH_HOSTNAME:=localhost} from ${SSH_BIND_IP:=0.0.0.0}:${SSH_TUNNEL_REMOTE:=${DEFAULT_PORT}} to ${SSH_TUNNEL_HOST=localhost}:${SSH_TUNNEL_LOCAL:=22}
```
Comment (jnovack) [inline entrypoint.sh]: In colloquial English, the words `over` and `from` change this sentence to mean the opposite of what it should.

The SSH_HOSTUSER@SSH_HOSTNAME is being tunneled OVER (more appropriately "inside") the REMOTE.

Please revert the words.

**Comment by Lukas0025** [comment]
I hope I fixed most of my mistakes in English. Unfortunately, English is not my primary language.

**Comment by jnovack** [comment]
> I hope I fixed most of my mistakes in English. Unfortunately, English is not my primary language.

Not a problem, I meant no offense. 

Just a reminder, you do have a conflict which is pending.

**Comment by jnovack** [comment]
Could you request this PR against `v2.0` rather than `master`?

**Comment by jnovack** [comment]
There's some major conflicts going on that will need to be resolved first.

**Comment by jnovack** [comment]
I will be unable to merge this until all conflicts are resolved.  

image

**Comment by Lukas0025** [comment]
I am confused. I rebase to v2.0 branch and with this branch I have no conflicts.

Snímek obrazovky_2020-08-17_10-31-27

**Comment by jnovack** [comment]
Whoa, that is interesting. I'm sorry, I blame Github.

Hmm, try this.  Let me **Close** this, then make another.

**Pull request closed without merging**